integrations/access/accessmonitoring/access_monitoring_rules.go

```diff
@@ -190,6 +197,20 @@ func (amrh *RuleHandler) RawRecipientsFromAccessMonitoringRules(ctx context.Cont
 	return recipientSet.ToSlice()
 }
 
+func (amrh *RuleHandler) getUserTraits(ctx context.Context, userName string) map[string][]string {
+	log := logger.Get(ctx)
+	const withSecretsFalse = false
+	user, err := amrh.apiClient.GetUser(ctx, userName, withSecretsFalse)
+	if trace.IsAccessDenied(err) {
+		log.WarnContext(ctx, "Missing permissions to read user.traits, please add user.read to the associated role", "error", err)
```

Comment:
Can we log the actual role name that should have `user.read` permissions?

Reply:
I'm going to defer this change.
The access-plugin role name is not available in this context. We'll need to modify the initialization of the app or extend the client interface to get the role name.